mcobs_eu_demo.postman_collection.json

```diff
@@ -0,0 +1,649 @@
+{
+	"info": {
+		"_postman_id": "b8cbacea-0202-4dd7-93fa-cde9faf00c27",
+		"name": "MCOBS EU API - demo",
```

Comment:
All collections shall have Mastercard at the beginning of the name.  Can you please unwrap MCOBS here and in other places.